KestrelMock.Tests/MockServiceTest.cs

```diff
@@ -4,22 +4,21 @@
 using Microsoft.AspNetCore.Mvc.Testing;
 using Microsoft.Extensions.Configuration;
 
-namespace KestrelMockServer.Tests
-{
-    public class MockTestApplicationFactory
+namespace KestrelMock.Tests;
```

Comment:
Maybe you'd prefer `KestrelMockServer.Tests`?  The main reason for the change was to bring them line with each other (project and namespace match) ... so we could go the other way?

Reply:
Looks more clear to me as KestrelMock.Tests